app/views/admin/hearings/_allocation_decision_fields.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class="nested-fields">
+  <div class="field mot-desc">
+    <%= f.hidden_field :hearing_id, value: @hearing.id %>
+    <%= f.label :motReasonDescription, "MOT Reason Description" %>
+    <%= f.select :motReasonDescription, options_for_select(ModeOfTrialReason.all.map{ |mot| [mot.reason_description,
```

Comment:
This has lost the blank option. Do you want to see if it's possible to put it back in? Only because the MotReasonCode relies on this field being changed to update.